servers/rendering/renderer_viewport.h

```diff
@@ -68,8 +68,7 @@ class RendererViewport {
 		Rect2 viewport_to_screen_rect;
 		bool viewport_render_direct_to_screen;
 
-		bool hide_scenario;
-		bool hide_canvas;
+		bool disable_2d;
```

Comment:
I guess to be consistent `disable_3d` should also be left uninitialized here and initialized when creating a struct.